CHANGES.rst

```diff
@@ -1,7 +1,45 @@
 Release Notes
 =============
 
-2.1.5
+2.1.7
+-----
+
+This release brings more robust control of the Pex cache (PEX_ROOT).
+
+The `--cache-dir` setting is deprecated in favor of build time control of the cache location with
+`--pex-root` and new support for control of the cache's runtime location with `--runtime-pex-root`
+is added. As in the past, the `PEX_ROOT` environment variable can still be used to control the
+cache's runtime location.
```

Comment:
If you know when you want to remove the deprecation, consider mentioning it. I imagine Pex 2.2.0?

Reply:
I don't, see: #584